internal/pkg/zip-package_test.go

```diff
@@ -31,6 +33,21 @@ func TestInstallPackage(t *testing.T) {
 	assert.Equal(t, 2, len(mf.Commands()))
 }
 
+func TestInstallPackageWithSetupError(t *testing.T) {
+	pkg, err := CreateZipPackage("assets/fake-wrong-setup-1.0.0.pkg")
+	assert.Nil(t, err)
+
+	target, err := os.MkdirTemp("", "cdt-package-test-*")
+	assert.Nil(t, err)
+
+	var previousValue = viper.GetBool(config.ENABLE_PACKAGE_SETUP_HOOK_KEY)
+	viper.Set(config.ENABLE_PACKAGE_SETUP_HOOK_KEY, true)
+	mf, err := pkg.InstallTo(target)
+	assert.NotNil(t, err)
```

Comment:
Can we make a more specific test, so that this is exactly the error we are expecting?